index.html

```diff
@@ -38,6 +38,29 @@ <h5 data-i18n="description:principles.software.title">Software lliure</h5>
 {% include sponsors.html %}
 {% include matchfunding_modal.html %}
 
+<div class="help-us call-to-action-block text-center">
+  <div class="row image-block">
+    <aside class="col-lg-6 offset-md-3">
+
+      <!-- Begin Mailchimp Signup Form -->
+      <div id="mc_embed_signup">
+        <form action="https://katuma.us3.list-manage.com/subscribe/post?u=c1fa2ee78d0632dc9cecbffab&amp;id=9a76bc5a8e" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+          <div id="mc_embed_signup_scroll">
+            <h2 data-i18n="newsletter.header"><label for="mce-EMAIL">Vols saber-ne més?</label></h2>
+            <p data-i18n="newsletter.subheader">Apunta't a la llista i estigues al dia</p>
+            <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email" required>
+            <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+            <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_c1fa2ee78d0632dc9cecbffab_9a76bc5a8e" tabindex="-1" value=""></div>
+            <input type="submit" value="Subscriu-t'hi" name="subscribe" id="mc-embedded-subscribe" class="button secondary-button">
```

Comment:
I think the only way to translate the input button is to use JS. So I propose to ship this as is and if we find a way, we can open a separate PR.